modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -533,7 +533,7 @@ public class IgniteImpl implements Ignite {
         ComponentWorkingDir partitionsWorkDir = partitionsPath(systemConfiguration, workDir);
 
         partitionsLogStorageFactory =
-                SharedLogStorageFactoryUtils.create("data log", clusterSvc.nodeName(), partitionsWorkDir.raftLogPath());
+                SharedLogStorageFactoryUtils.create("table data log", clusterSvc.nodeName(), partitionsWorkDir.raftLogPath());
```

Comment:
table or partition? We usually use the latter, but IDK which is better here

Reply:
Well... depends on a level of abstraction that you choose. That's for us anyway